### ISSUE TITLE
Hangar load does not always trigger gridbackup it seems

### ISSUE THREAD

**Issue by kontu** [Casimir255/QuantumHangar#29]
Hangar load does not always trigger gridbackup it seems

Player loaded these 4 grids a few minutes before a crash (so they were lost in the world as expected); they had been manually imported several weeks ago with hangar sync
``` [33] - #05-AngryBee_08.30.2020_10.53.22
 [34] - #06-AngryBee_08.30.2020_13.11.04
 [35] - #07-AngryBee_08.30.2020_13.24.57
 [36] - #08-AngryBee_08.30.2020_13.35.24
```
But there's no Gridbackup triggered for them anywhere

Now here's where it gets *really fun*. I dug into the daily backups and grabbed copies of these grids.

Looking at lots of other blueprints from other peoples garages; they all have this value set:
      <OwnerSteamId>0</OwnerSteamId>

For whatever reason, #05 bee has:
<OwnerSteamId>90138397116871689</OwnerSteamId>
and 6,7,8 have
<OwnerSteamId>90138402485211145</OwnerSteamId>

Those aren't even SteamID's...Searching those numbers in the Sandbox gets 0 hits at all. I have no fucking clue where they've come from but I suspect whatever it is is what caused the backups not to work?
All blocks with ownership are properly marked as owned by the players IdentityID...I don't even know what to make of this hah. These grids were only ever saved by Quantum Hangar (never any other garage type system); so I have no clue what to do about this

**Comment by Casimir255** [comment]
So the 90138397116871689 are actually NPC identities. Meaning the player didn't fully owner the grid, or wasn't the big owner. I think im attempting to backup the grids that people attempt to hanger, but hanger refuses load later after the load. Same with 0.

Ultimately I'm backing up the grid too early

**Comment by kontu** [comment]
Interesting. Nearly all grids have 0 as the SteamIDOwne (I checked around 200 from various different hangars)r; and when I searched on "Owner" for the grid; they owned every single block with ownership. 

I did some more digging there *is* a small chance it was originally under a different plugin. I had them load and save it again and the ID is now 0; and future save/loads seem to be creating gridbackups fine on save and load
